contributors/devel/api-conventions.md

```diff
@@ -306,34 +306,49 @@ response reduces the complexity of these clients.
 ##### Typical status properties
 
 **Conditions** represent the latest available observations of an object's
-current state. Objects may report multiple conditions, and new types of
-conditions may be added in the future. Therefore, conditions are represented
-using a list/slice, where all have similar structure.
+state.  They are an extension mechanism intended to be used when the details of
+an observation are not a priori known or would not apply to all instances of a
+given Kind.  For observations that are well known and apply to all instances, a
+regular field is preferred.  An example of a Condition that probably should
+have been regular field is Pod's "Ready" condition - it is managed by core
+controllers, it is well understood and applies to all Pods.
```

Comment:
probably worth noting that it is difficult/impossible to change the factors that combine to determine a single field value. For example, pod readiness could not decide in the future to also include other factors like service endpoint propagation, etc, without potentially breaking API clients that depended on a particular meaning of pod readiness.

Reply:
Interesting choice, since we are, in fact, changing the factors that combine to define pod readiness :)

I'll take a stab at words.